hack/get-kube-binaries.sh

```diff
@@ -10,7 +10,7 @@ set -u
 # The integration test framework expects these binaries to be found in the PATH.
 
 # This is the kube-apiserver version to test against.
-KUBE_VERSION="${KUBE_VERSION:-v1.11.3}"
+KUBE_VERSION="${KUBE_VERSION:-v1.18.8}"
```

Comment:
The api version `apiextensions.k8s.io/v1` is only available from 1.16

Reply:
could you change to 1.17.0 as in https://github.com/metacontroller/metacontroller/pull/49